assignments/closure.js

```diff
@@ -17,4 +28,13 @@ const counterFactory = () => {
   // Return an object that has two methods called `increment` and `decrement`.
   // `increment` should increment a counter variable in closure scope and return it.
   // `decrement` should decrement the counter variable and return it.
-};
+
+  let count = 0;
+  console.log(count);
+  let add = () => count +=1;
+  let sub = () => count -=1;
```

Comment:
looks like you closed a function you didn't start here. You have the object that increments and decrements, but you don't have the function. I am assuming the start was lost in the copy paste back from code pen